input/resources/au-core-requester.xml

```diff
@@ -970,6 +976,8 @@
     <div>
       <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
       <p>The responder <strong>SHALL</strong> support both.</p>
+      <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```

Comment:
This should be **SHALL** 
See AU Core Encounter notes:
SHALL support _[multipleOr](http://hl7.org/fhir/R4/searchparameter-definitions.html#SearchParameter.multipleOr)_ search on status (e.g.status={system|}[code],{system|}[code],...)

Reply:
fixed

Reply:
fixed